acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -85,7 +85,7 @@
           sleep $sleep_for
           url='https://#{master}:8140/puppet/v3/catalog/#{agent_cert}?environment=production'
           echo "Curling: $url"
-          curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
+          env PATH='#{agent[:privatebindir]}:${PATH}' curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
```

Comment:
Hm, so @joshcooper do you think there's a safer way, or are we stuck on `privatebindir`since it is one of the few places to find the bindir? I noticed that other tests use `privatebindir` whenever we need to find our ruby, so maybe this is the best option we have?